.changeset/loud-badgers-fly.md

```diff
@@ -0,0 +1,31 @@
+---
+"@frontify/guideline-blocks-settings": minor
+---
+
+refactor(LinkSelector): Removed `appBridge` prop in favor of directly passing document handling functions: `getAllDocuments`, `getDocumentPagesByDocumentId`, and `getDocumentSectionsByDocumentPageId` for clearer API and enhanced modularity
+
+Replace this:
+
+```jsx
+<LinkSelector
+    url={url}
+    appBridge={appBridge}
+    onUrlChange={onUrlChange}
+    buttonSize={buttonSize ?? ButtonSize.Medium}
+/>
+```
+
+with:
+
+```jsx
+<LinkSelector
+    url={url}
+    onUrlChange={onUrlChange}
+    buttonSize={buttonSize ?? ButtonSize.Medium}
+    getAllDocuments={appBridge.getAllDocuments}
```

Comment:
`bind` missing from the changeset?

Reply:
improved the syntax so we don't need the `bind` and adjusted it here